src/dsp_tools/resources/schema/data.xsd

```diff
@@ -576,8 +576,10 @@
     <xs:element name="knora">
         <xs:complexType>
             <xs:sequence>
-                <xs:element name="permissions" type="permissions_type" minOccurs="0" maxOccurs="unbounded"/>
-                <xs:element name="authorship" type="authorship_type" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:choice maxOccurs="unbounded">
+                    <xs:element name="permissions" type="permissions_type" minOccurs="0" maxOccurs="unbounded"/>
+                    <xs:element name="authorship" type="authorship_type" minOccurs="0" maxOccurs="unbounded"/>
+                </xs:choice>
```

Comment:
My script writes the `<authorship>` tags at the start of the file, which was not allowed by the XSD. There were 2 possible solutions:

1. Write the `<authorship>` tags after the `<permissions>` tags.
2. Relax the restriction of the XSD, and allow the `<authorship>`and `<permissions>` tags to be freely ordered (and mixed).

I chose (2) because I don't think we should have unnecessary restrictions.